device/cuda/src/cca/component_connection.cu

```diff
@@ -963,7 +963,7 @@ __global__ void partition_sorting_kernel(unsigned* out,
 }
 
 std::tuple<vecmem::unique_alloc_ptr<unsigned[]>, std::size_t> partition_gpu(
-    const cell_container_types::host& data, vecmem::memory_resource& mem,
+    const cell_container_types::host& /*data*/, vecmem::memory_resource& mem,
```

Comment:
I don't know why the same error didn't occur in the previous PR. Maybe it is because I updated the detray version?

```
__w/traccc/traccc/device/cuda/src/cca/component_connection.cu: In function 'std::tuple<std::unique_ptr<unsigned int [], vecmem::details::unique_alloc_deleter<unsigned int []> >, long unsigned int> traccc::cuda::details::partition_gpu(const host&, vecmem::memory_resource&, traccc::cuda::details::cell_container)':
/__w/traccc/traccc/device/cuda/src/cca/component_connection.cu:966:1: error: unused parameter 'data' [-Werror=unused-parameter]
  965 | std::tuple<vecmem::unique_alloc_ptr<unsigned[]>, std::size_t> partition_gpu(
      |                                                                             
  966 |     const cell_container_types::host& data, vecmem::memory_resource& mem,
      | ^   
cc1plus: all warnings being treated as errors
make[2]: *** [device/cuda/CMakeFiles/traccc_cuda.dir/build.make:188: device/cuda/CMakeFiles/traccc_cuda.dir/src/cca/component_connection.cu.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:4079: device/cuda/CMakeFiles/traccc_cuda.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

Reply:
> I'm not sure how this wasn't triggered by the CI in https://github.com/acts-project/traccc/pull/309

Yeah that's what I meant. It should have been reported by #309 unless any change in this PR is triggering it. hmm,.